Encounter Me.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.31624.102
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Encounter Me", "Encounter Me\Encounter Me.csproj", "{6CDC5C78-FB8B-4413-BA5C-7203A2DA8453}"
```

Comment:
What's up with this change?

Reply:
I am not sure myself - I think it vas VS doing automatic refactoring as I changed project structure. I added packages, then deleted some also created/deleted some files, moved classes to different folders.

Reply:
I am not sure myself - I think it vas VS doing automatic refactoring as I changed project structure. I added packages, then deleted some also created/deleted some files, moved classes to different folders.